processor/metric/package_tests/TestProcessMetricMinimal.approved.json

```diff
@@ -0,0 +1,30 @@
+{
+    "events": [
+        {
+            "@timestamp": "2017-05-30T18:53:42.281Z",
+            "context": {
+                "service": {
+                    "agent": {
+                        "name": "elastic-go",
+                        "version": "0.1.0"
+                    },
+                    "name": "apm-server"
+                }
+            },
+            "metric": {
```

Comment:
Our convention is that "minimal" payloads are only have the required attrs, in this case would be just the gauge or the counter metric, but not the 2.
Look eg. for errors with have a minimal example with an exception and another with a log

Reply:
fair enough, I'll trim it further.

Reply:
done